README.md

```diff
@@ -272,9 +275,6 @@ sync_users(storage_id="storage_di", action="action")
 # Get client role id from name
 role_id = keycloak_admin.get_client_role_id(client_id=client_id, role_name="test")
 
-# Get all roles for the realm or client
-realm_roles = keycloak_admin.get_roles()
```

Comment:
I believe the possibility to get realm roles is still available, it's merely just called `keycloak_admin.get_realm_roles()`. Perhaps we should keep this in the README examples, just fix the method name? :)

Reply:
@ryshoooo I think you skipped reading my "Note" section in my upper comment 😄

I meant keycloak_admin.get_roles() method doesn't exist, but keycloak_admin.get_realm_roles() exists, and in the README we have both documented which is wrong, so I deleted the documentation for the keycloak_admin.get_roles(), and the other is still there.

The method name seems correct, and I think it should remain keycloak_admin.get_realm_roles(), because we getting the roles for only one realm of a Keycloak server which is defined upon initializing KeycloakAdmin class.